game/scripts/npc/abilities/luna_eclipse.txt

```diff
@@ -25,22 +25,22 @@
 		"AbilityCastPoint"				"0.6 0.6 0.6 0.6 0.6"
 		"AbilityCastRange"				"0"
 
-		// Time		
+		// Time
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityCooldown"				"140.0"
 
 		// Cost
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityManaCost"				"150 200 250 350 750"
-		
+
 		// Special
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilitySpecial"
 		{
 			"01"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"beams"						"5 8 11 17 40"
+				"beams"						"5 8 11 13 15"
```

Comment:
I'm also in favour of 14 and 17 so that it keeps being +3 per rank. I don't think the 2 extra beams will break everything again.